### ISSUE TITLE
Not working if site runs under sub-directory

### ISSUE THREAD

**Issue by YasserAAA** [Shazwazza/Smidge#95]
If your site address like this: "http://RootDmain.com/YoursiteApp", you will have a problem when composing bundles.

This I fixed this issue in **"FileSystemHelper.cs"** as follow:
in function **"IFileInfo GetFileInfo(IWebFile webfile)"**, add this line after the first statement:
`path = path.Substring(_siteInfo.GetBasePath().Length);`

the whole function will be like:

```
public IFileInfo GetFileInfo(IWebFile webfile)
        {
            var path = webfile.FilePath.TrimStart(new[] { '~' });
            path = path.Substring(_siteInfo.GetBasePath().Length);
            var fileInfo = _fileProvider.GetFileInfo(path);

            if (!fileInfo.Exists)
            {
                throw new FileNotFoundException($"No such file exists {fileInfo.PhysicalPath} (mapped from {path})", fileInfo.PhysicalPath);
            }

            return fileInfo;
        }
```

**"_siteInfo"** of course must be injected from **IWebsiteInfo**.

**Comment by Shazwazza** [comment]
Great thanks for the help! Any chance you have time to create a Pull Request for the above?

**Comment by YasserAAA** [comment]
Sorry for delay. I'm afraid I have no time these days. 

I appreciate your nice library.

**Comment by Shazwazza** [comment]
I know this is super old, sorry about that, just haven't had much time to look into these issues. I've just pushed a fix and will ship a 3.2 release shortly.
